### ISSUE TITLE
Rework Test Application using Entity Class

### ISSUE THREAD

**Issue by NelsonNew** [devonfw/cobigen#1580]
As a user, I want to test cobigen code generation using an entity class with the following tutorial:
https://github.com/devonfw/cobigen/wiki/howto-devonfw-ide-CobiGen-PoC-E2E

The tutorial should show a step-by-step process using the current version of cobigen instead of an old version.

**Comment by leonie19s** [comment]
We suggest fully reworking the wiki tutorial as it is too extensive with only a little focus on the CobiGen part. Moreover, it is hard to follow, as the descriptions and images provided no longer fit, since the devon version is outdated. 
Shortening the tutorial would be helpful to shift the focus on the functionalities of CobiGen. We would also introduce the CLI  to the user, which was not mentioned before.
